packages/suite/src/views/dashboard/components/DashboardPassphraseBanner.tsx

```diff
@@ -0,0 +1,58 @@
+import { selectDevice } from '@suite-common/wallet-core';
+import { Warning, H3, Text, Button, IconButton, Row, Column } from '@trezor/components';
+import { goto } from 'src/actions/suite/routerActions';
+import { setFlag } from 'src/actions/suite/suiteActions';
+import { Translation } from 'src/components/suite';
+import { SettingsAnchor } from 'src/constants/suite/anchors';
+import { useDiscovery, useDispatch, useSelector } from 'src/hooks/suite';
+import { selectSuiteFlags } from 'src/reducers/suite/suiteReducer';
+
+export const DashboardPassphraseBanner = () => {
+    const dispatch = useDispatch();
+    const { isDashboardPassphraseBannerVisible } = useSelector(selectSuiteFlags);
+    const device = useSelector(selectDevice);
+    const { isDiscoveryRunning } = useDiscovery();
+
+    if (
+        isDashboardPassphraseBannerVisible === false ||
+        device?.useEmptyPassphrase === true ||
```

Comment:
I think the logic here shall be different :thinking: maybe?

I think, if all your devices are `useEmptyPassphrase===false`  the we shall show banner? Not sure if this is enough, maybe lets discuss this. 

This way the banner will be visible in EVERY passphrase wallet regardles if user uses the default one as well. Is this intended?

Reply:
I don't think that it should be only when all have `useEmptyPassphrase===false`

But we need @Hermez-cz for decide this

Anyway, we can merge it this way and update it later. I also added another condition so the banner will not be visible anymore when your default wallet is already passphrase.